### ISSUE TITLE
 predict & target shape do not match 

### ISSUE THREAD

**Issue by ll-gh** [MontaEllis/Pytorch-Medical-Segmentation#54]
How to solve this problem?
input size:torch.Size([2, 2, 32, 32, 32]) target size: torch.Size([2, 64, 32, 32])

**Comment by BelieferQAQ** [comment]
how to solve this problem？

**Comment by QianLingjun** [comment]
我是改成这样子，就可以了。loss = criterion_ce(outputs, target_argmax) + criterion_dice(outputs_argmax, target_argmax)
